include/wx/generic/gridsel.h

```diff
@@ -19,7 +19,6 @@
 
 #include "wx/vector.h"
 
-typedef wxVector<wxGridBlockCoords> wxVectorGridBlockCoords;
```

Comment:
Not sure how did we end up with duplicate `wxGridBlockCoordsVector` and `wxVectorGridBlockCoords` typedefs, but unfortunately we can't just remove it from the public header any longer, even if it's clearly not needed.

Instead, it should be deprecated, with the message saying to use `wxGridBlockCoordsVector` instead, but left here (in a separate commit).